QuestionBot/QuestionBot/Model/QuestionMessageListener.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuestionBot.Model{
+
+    public class QuestionMessageListener : IMessageListener{
+
+        private IStore _questionDataStore;
+
+        public QuestionMessageListener(IStore store){
+            _questionDataStore = store;
+        }
+
+        public void ReceiveMessage(string input){
+            const string questionCommand = "/question";
+            const string answerTag = "/answer";
+
+            if (input.StartsWith(questionCommand)){
+                string questionText = input.Remove(0, questionCommand.Length);
+                questionText = questionText.TrimStart(' ');
+
+                if (!String.IsNullOrWhiteSpace(questionText)){
+                    IRecord newQuestion = _questionDataStore.CreateRecord(questionText);
+                    Console.WriteLine("Question has been created with ID " + 
+                        newQuestion.Id + 
+                        ". Question: " +
+                        newQuestion.Question);
+                }else{
+                    Console.WriteLine("This question appears to be blank, please retry.");
+                }
+
+            }else if (input.StartsWith(answerTag)){
```

Comment:
This section is not part of the requirements. The plan is to create another `IMessageListener`, `AnswerMessageListener` to handle such messages.

>  `QuestionMessageListener.ReceiveMessage` should listen for messages of the form `/question <question>`